python/src/scipp/show.py

```diff
@@ -114,8 +110,7 @@ def _events_extent(self):
             # but may have not data
             # Find a events coord to use for determining length
             for coord in self._variable.coords.values():
-                if sc.contains_events(coord):
-                    data = coord.values
+                pass
```

Comment:
If we are just passing don't need this loop at all. Can do so in the context above.